src/pages/mypage/components/missioncard/MissionList.tsx

```
@@ -0,0 +1,141 @@
+    if (timeSort === "short")
+      copied.sort((a, b) => a.expectedMinutes - b.expectedMinutes);
+    if (timeSort === "long")
+      copied.sort((a, b) => b.expectedMinutes - a.expectedMinutes);
+    if (timeSort === "recent") {
+    }
+
+    return copied;
+  }, [baseList, timeSort, category]);
+
+  useEffect(() => {
+    setVisibleCount(loadSize);
```

Biome (2.3.13)

[error] This hook specifies more dependencies than necessary: list.length, loadSize.

This dependency can be removed from the list.

This dependency can be removed from the list.

React relies on hook dependencies to determine when to re-compute Effects.
Specifying more dependencies than required can lead to unnecessary re-rendering
and degraded performance.
Unsafe fix: Remove the extra dependencies from the list.


(lint/correctness/useExhaustiveDependencies)